addOns/dev/src/main/zapHomeFiles/dev-add-on/html/localStorage/jwt-none.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>ZAP Test Server</title>
+	<link href="/tutorial.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<div class="roundContainer">
+	<h1>ZAP Test Server</H1>
+	<h2>JWT with None algorithm in localStorage</h2>
+
+	<script>
+	localStorage.setItem('token', 'eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.');
+	</script>
+	
+	Data stored in localStorage.
+	
+	<ul>
+	<li><a href="./">..</a>
```

Comment:
.. suggests going up/back but ./ doesn't.

Reply:
Same in others

Reply:
It does, because the page is `localStorage/jwt-none.html` and `./` resolves to `localStorage`.
Using `..` would go 2 levels up.
Try it out 😁